assets/styles/common/box-form-container.scss

```diff
@@ -129,23 +126,63 @@
 }
 
 .provider-google {
-  color: white !important;
-  background-color: #ea4335 !important;
+  font-family: 'Roboto', sans-serif !important;
+  color: $google-button-font-color !important;
+  background-color: $google-button-background-color !important;
+  padding: 0 !important;
 
   &.btn {
     &:hover {
-      background-color: darken(#ea4335, 10%) !important;
+      background-color: darken($google-button-background-color, 3%) !important;
+    }
+  }
+
+  .google-auth-container{
+    display: flex;
+    align-items: center;
+
+    .google-icon-container{
+      padding-right: 6px;
```

Comment:
Is there a possibility of being consistent and use `em` and `rem` always? (maybe it's too hard to change)